.travis.yml

```diff
@@ -14,27 +14,36 @@ branches:
 
 sudo: true
 
-php:
-  - 7.0
-  - 7.1
-  - hhvm
-
 matrix:
   fast_finish: true
   include:
+    - php: 7.0
+      env: ES_VERSION="5.5" TEST_BUILD_REF="origin/5.5"
+    - php: 7.0
+      env: ES_VERSION="6.0" TEST_BUILD_REF="origin/6.0"
+    - php: 7.0
+      env: ES_VERSION="6.x" TEST_BUILD_REF="origin/6.x"
+
     - php: 7.1
-      env: ES_VERSION="5.0" TEST_BUILD_REF="origin/5.0"
+      env: ES_VERSION="5.5" TEST_BUILD_REF="origin/5.5"
     - php: 7.1
-      env: ES_VERSION="5.1" TEST_BUILD_REF="origin/5.1"
+      env: ES_VERSION="6.0" TEST_BUILD_REF="origin/6.0"
     - php: 7.1
-      env: ES_VERSION="5.2" TEST_BUILD_REF="origin/5.2"
+      env: ES_VERSION="6.x" TEST_BUILD_REF="origin/6.x"
+
+    - php: hhvm
+      env: ES_VERSION="5.5" TEST_BUILD_REF="origin/5.5"
+    - php: hhvm
+      env: ES_VERSION="6.0" TEST_BUILD_REF="origin/6.0"
+    - php: hhvm
+      env: ES_VERSION="6.x" TEST_BUILD_REF="origin/6.x"
+
   allow_failures:
     - php: hhvm
+    - env: ES_VERSION="6.x" TEST_BUILD_REF="origin/6.x"
```

Comment:
This is related to my previous comment:
> Maybe the Travis configuration can be changed so it is always running the builds against specific version (so the tests won't start failing after the new version is released). <-- What do you think?

Running against `6.x` means that the build can start failing without any changes in `elasticsearch-php` (because something was changed in the Elasticsearch itself).

I suggest the following:
- let the build run against `ES_VERSION="6.x"`
- put it into `allow_failures` so it won't break the build unexpectedly 
- when the new ES version is branched, it should be added to the build matrix (and checked that the build passes)